proxy/core/src/main/scala/io/cloudstate/proxy/HttpApi.scala

```diff
@@ -517,35 +523,49 @@ object HttpApi {
         )
 
       if (methDesc.isServerStreaming) {
-        def toChunk(entityMessage: MessageOrBuilder): HttpEntity.Chunk =
-          HttpEntity.Chunk(
-            if (isHttpBodyResponse) extractDataFromHttpBody(entityMessage) ++ NEWLINE_BYTES
-            else ByteString(jsonPrinter.print(entityMessage)) ++ NEWLINE_BYTES
-          )
+        val sseAccepted = new MediaTypeNegotiator(grpcRequest.headers).isAccepted(MediaTypes.`text/event-stream`)
```

Comment:
@raboof Is there a more efficient way of doing this?

Reply:
I guess the first question is whether this is the behavior you want: if a client sends `Accept: application/grpc,*/*;q=0.1`, do you want to send it SSE?

Reply:
@raboof What I want is that if the client expressedly wants text/event-stream, then that is what they'll get. How'd I express that?

Reply:
something like `grpcRequest.header[Accept].exists(_.mediaRanges.exists(r => r.mainType == "text" && r.subType == "event-stream"))`?

Reply:
@raboof Would this variant do the trick?

```
grpcRequest.header[Accept].exists(_.mediaRanges.exists(_.matches(MediaTypes.`text/event-stream`)))
```

Reply:
I don't think that will do what you want, as the media range `*/*` will match `text/event-stream`.

I see `MediaRange` doesn't expose a `subType`. It looks like `grpcRequest.header[Accept].exists(_.mediaRanges.exists(_.value.startsWith("text/event-stream")))` might fit?